sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+            - Select LC TAP via strap pins.
+            - Request transition to TEST_UNLCOKED by providing the TEST_UNLOCK_TOKEN provisioned
+              in previous steps.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc_state.
+
+            Repeat the same flow for the number of supported TEST_LOCK/UNLOCKED states.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_exit_token
+      desc: '''Verify provisioning and use of test exit token.
+
+            The test exit token is used to move the device from test mode into DEV, PROD or
+            PROD_END state. This test also incorporates enabling of ROM execution before
+            triggering the lc_state change.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Program TEST_EXIT_TOKEN in OTP transition via JTAG.
+            - Reset device by toggling POR pin.
+            - Select CPU JTAG via strap pins.
+            - Program CREATOR_SW_CFG_ROM_EXEC_EN in OTP via JTAG.
+            - Select LC TAP via strap pins (at runtime).
+            - Request transition into {pick one: DEV, PROD, PROD_END} state providing TEST_EXIT_TOKEN.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc state.
+
+            After this test most of the following manuf_ft_* test cases can be loaded into flash via
+            boostrap in silicon.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_sku_individualization
+      desc: '''Configure device fuses.
+
+            This includes configuration for most countermeasures and hardware options. It does not
+            include configuration of the device identity and RMA token, which is done in the
+            personalization step.
+
+            The test steps are expected to be executed from flash. In silicon, the program is loaded
+            using the ROM bootstrap protocol. It is recommended to use the same approach on the FPGA,
+            and enable it as a configuration option in DV.
+
+            - Pre-load OTP with {pick one: DEV, PROD, PROD_END} lc_state in pre-silicon
+              environments. Advance to state in silicon.
+            - Load test program into flash and start execution.
+            - Check device lc_state. Return error to the host if device in invalid state.
+            - Configure the following OTP fuses to enable access to entropy:
+              - EN_CSRNG_SW_APP_READ
+              - EN_ENTROPY_SRC_FW_READ
+              - EN_ENTROPY_SRC_FW_OVER
+            - Configure remaining HW_CFG OTP fields:
+              - DEVICE_ID
+              - MANUF_STATE
```

Comment:
`MANUF_STATE`: Need to check when the Silicon Creator is planning to write to this word.

Reply:
noting my comment in the meeting here: We also have `manuf_state_creator` and `manuf_state_owner` as part of usage constraints, compared with `CREATOR_SW_CFG_MANUF_STATE` and `OWNER_SW_CFG_MANUF_STATE`, respectively.